llvm-runtimes/libcxx/libcxx-20.1.7.ebuild

```diff
@@ -13,7 +13,10 @@ HOMEPAGE="https://libcxx.llvm.org/"
 LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
 SLOT="0"
 KEYWORDS="amd64 arm arm64 ~loong ~riscv sparc x86 ~arm64-macos ~x64-macos"
-IUSE="+clang +libcxxabi +static-libs test"
+IUSE="
+	+clang +libcxxabi +static-libs test
+	default-compiler-rt default-libcxx default-lld
```

Comment:
I don't think this is a good idea. These flags exist precisely as a convenience to control the *default* configs for new clang installs. Using them to override the config implies precisely that users can no longer rely on their config being respected.

Reply:
Well, the same flags are used also for llvm-core/clang, before the config is made llvm-core/clang-runtime, so I would argue that these flags are already used for bootstraping parts of LLVM toolchain, before the config is assembled. These flags are usually enforced by profiles and not set by people manually.

But to not ignore the configuration, I could first check if the config exist and fall back to overwriting LDFLAGS if it doesn't.

Do you have any other alternatives in mind? The only one which I could think of is falling back to old clang if the config is not present. But I feel that's worse - I think we should aim to use the new clang as soon as it's available, even though we are still assembling its runtime.